recipe/bld.bat

```diff
@@ -1,36 +1,51 @@
 setlocal EnableDelayedExpansion
 
-copy %RECIPE_DIR%\CMakeLists.txt .\CMakeLists.txt
+if %ARCH%==32 (
+	set PLATFORM=Win32
+) else if %ARCH%==64 (
+	set PLATFORM=x64
+)
 
-:: Make a build folder and change to it.
-mkdir build
-cd build
+:: See https://github.com/conda-forge/staged-recipes/pull/194#issuecomment-203577297
+:: Nasty workaround. Need to move a more current msbuild into PATH.  The one on
+:: AppVeyor barfs on the solution. This one comes from the Win7 SDK (.net 4.0),
+:: and is known to work.
+if %VS_MAJOR%==9 (
+    set "PATH=C:\Windows\Microsoft.NET\Framework\v4.0.30319;%PATH%"
+)
 
-:: Configure using the CMakeFiles
-cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" -DCMAKE_BUILD_TYPE:STRING=Release .. 
```

Comment:
I don't understand much about building on Windows but does the `cmake` alternative still exists? It is usually easier to debug.

Reply:
I composed that CMakeLists.txt a long time ago, and never updated it to support the newer version of sdl2_mixer. The general concensus of the SDL2 module maintainers is that windows builds are already accounted for by the included .sln files, and therefore they don't feel much for also supporting the cmake build system.

Reply:
Got it. Sorry but I can't really help you much there. Not sure if the `vc` activation are to blame though. Maybe @isuruf has a better insight.